modules/core/src/main/java/org/apache/ignite/internal/binary/BinaryObjectEx.java

```diff
@@ -38,4 +38,11 @@
      * @throws BinaryObjectException If failed.
      */
     @Nullable public BinaryType rawType() throws BinaryObjectException;
+
+    /**
+     * Check if flag set.
```

Comment:
There should be a carriage return symbol between method's description and the first @param specified.